TanStack/virtual
# fix(examples): Remove conflicting React type definitions

## What this PR solves
- It solves the `jsx` type error problem in certain examples of official documents.
<img width="1912" alt="스크린샷 2024-07-29 오후 6 16 28" src="https://github.com/user-attachments/assets/f66f63e9-36b4-4a8c-a005-a38fe981ff41">

### Type Error List 
- [fixed](https://tanstack.com/virtual/latest/docs/framework/react/examples/fixed)
- [dynamic](https://tanstack.com/virtual/latest/docs/framework/react/examples/dynamic)
- [table](https://tanstack.com/virtual/latest/docs/framework/react/examples/table)
- [window](https://tanstack.com/virtual/latest/docs/framework/react/examples/window)

## Summary
- I think the problem is that the type definition is duplicated.

## Steps to Reproduce
1. Go to the example project page of the official document created above.
2. The package goes down and waits to run.
3. Check the type error in the `jsx` part.

## How to Test the Solution
``` 
git clone https://github.com/TanStack/tanstack.com.git

cd ./tanstack.com

Go to the file 
- app/routes/$libraryId.$version.docs.framework.$framework.examples.$.tsx

Change the value of the 'stackBlitzUrl' variable to 
  'https://stackblitz.com/github/SaeWooKKang/virtual/tree/main/examples/react/fixed?embed=1&theme=light&preset=node&file=src/main.tsx'.

pnpm install

pnpm dev

Check example in browser
```
